src/taker/api.rs

```diff
@@ -12,15 +12,14 @@ use std::{
     collections::{HashMap, HashSet},
     io::BufWriter,
     net::TcpStream,
-    path::{Path, PathBuf},
-    process::Child,
+    path::PathBuf,
     thread::sleep,
     time::{Duration, Instant},
 };
 
 use bitcoind::bitcoincore_rpc::RpcApi;
 
-#[cfg(feature = "tor")]
+#[cfg(not(feature = "integration-test"))]
```

Comment:
I think we can now safely remove all these feature gates. Now `ConnectionType::Tor` doesn't have to be gated behind feature tor. Tor is available by default to all builds, and switched on/off via the connection type enum. 

All tests already sets it to clearnet. And the default for all apps should be tor. 

This will remove these awkward gates. great riddance.

Reply:
Everywhere there is a `#[cfg(not(feature = "integration-test"))]` that can be removed i think.  Also will allow you to remove some `#[allow(unused_variables)]`s.